src/content/article/over-a-year-late-a-dollar-short-more-than-two-decades-of-kodi/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Over a Year Late, A Dollar Short - More Than Two Decades of Kodi!
+date: 2024-03-24T11:45:55.334Z
+author: Keith Herrington
+tags:
+  - Community
+featured_image:
+  src: /images/blog/xbmc-2003-1080p.webp
+  title: XBOX Media Player Home Screen, 2003
+  alt: Image of the original XBMP home screen and menu
+---
+Hi, everyone - I'm Keith, and while I now find myself President of the Kodi Foundation, my initial involvement with Kodi goes way back, as a humble user. Since our 20th anniversary came and went, I've been meaning to write a bit of a look back on how we got here - how *I* got here. This is the story of my journey.
+
+**In the Beginning...**
+
+In 2002, a couple guys saw the potential of being able to watch media on their own terms, through what was a new type of device: a "connected" TV device. They wanted to bring many things you'd traditionally need a computer for to the living room, operating it from the couch without needing a keyboard or mouse, controlling events through what they called a *10 foot interface* because, well, it was an interface you could sit 10 feet away from.
```

Comment:
```suggestion
In 2002, a couple of guys saw the potential of being able to watch media on their own terms, through what was a new type of device: a "connected" TV device. They wanted to bring many things you'd traditionally need a computer for to the living room, operating it from the couch without needing a keyboard or mouse, controlling events through what they called a *10 foot interface* because, well, it was an interface you could sit 10 feet away from.
```

Reply:
US vernacular - the voice of the author - so I left this "as is".